.github/workflows/main.yml

```diff
@@ -182,8 +182,11 @@ jobs:
       extra_envs: -e TORCH_BLADE_BUILD_WITH_CUDA_SUPPORT=OFF
         -e TORCH_BLADE_CI_BUILD_TORCH_VERSION=1.8.1+cpu
       exec_command: bash ./scripts/ci/test_pytorch_blade.sh
-  NGC-TORCH-1110:
-    uses: ./.github/workflows/ngc_reusable.yml
+    secrets:
```

Comment:
Shall add secrets for `NGC-TORCH-1110`

Reply:
We don't need to trigger the NGC job on the merged event, so I remove the `NGC-TORCH-1110` job at L185. If we only need the cronjob, just keep the `ngc_cronjob.yaml`.